docs/website/docs/walkthroughs/zendesk-weaviate.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Importing Ticket Data from Zendesk API to Weaviate'
+description: Importing Ticket Data from Zendesk API to Weaviate
+keywords: [how to, zendesk, weaviate, vector database, vector search]
+---
+
+# Importing Ticket Data from Zendesk API to Weaviate
+
+Zendesk is a cloud-based customer service and support platform. Zendesk Support API, which is also known as the Ticketing API lets’s you access support tickets data. By analyzing this data, businesses can gain insights into customer needs, behavior, trends, and make data-driven decisions. The newest type of databases, vector databases, can help in advanced analysis of tickets data such as identifying common issues and sentiment analysis.
+
+In this walkthrough, we’ll show you how to import Zendesk ticket data to one of the vector databases, Weaviate. We’ll use dlt to connect to the Zendesk API, extract the data, and load it into Weaviate.
```

Comment:
Be more specific: the point of the walkthrough is to import tickets data (like user name, date status etc.) and text content (like "description") in order to ask questions about the tickets (ie. what is the most common problem our user have?)

I'm lacking information that the pipeline loads data incrementally (new tickets) and is also able to update tickets that changed (merge) and it all works with Waviate (you can do it here or at the end) - so we have the same behavior as in relational databases

Reply:
> ask questions about the tickets (ie. what is the most common problem our user have?)
Asking questions (retrieval augmented generation) was out of scope for original purpose of this walkthrough. Weaviate now has [generative search](https://weaviate.io/developers/weaviate/search/generative) feature for RAG. I can extend the examples to include this as well.